scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -426,7 +426,7 @@ class FormatOps(
       if (groups.lengthCompare(1) == 0)
         // for the last group, return '{' or ':'
         findTreeInGroup(groups.head, func) { x =>
-          templateCurly(template).getOrElse(tokenAfter(x).left)
+          templateCurly(template).getOrElse(getLastNonTrivialToken(x.last))
```

Comment:
@tgodzik when is this triggered? what does the test result look like without it?

Reply:
It would set the expire at the comment:
```
trait A

object Foo extends A
object Bar
    extends A

    /** A doc comment */
object D extends A
```

This makes the extend go over the next line and the comment indented.

Reply:
Maybe it's fine to leave it here? Since it's an overall utility method? But I can also just inline it if you'd prefer it.